client/ayon_xstudio/plugins/load/open_in_xstudio.py

```diff
@@ -93,10 +97,8 @@ def load(
             # with 0. "1001" has a padding of 0 and "0101" has a padding of 1.
             # we output the seq path with hashes.
             idxs = list(seq.indexes)
-            pad = "#" * (seq.padding + len(str(idxs[0])))
-            first_image = (
-                f"{seq.head}{pad}{seq.tail}={idxs[0]}-{idxs[-1]}"
-            )
+            pad = "#" * len(str(idxs[-1]))
```

Comment:
Maybe it was already discussed but isn't this causing possible issue? If I would have frames 1-10, then this would be `##`, but the frame string in filename is `0010`, so it should be `####`. Which I belive the previous logic did handle.